Formula/filebeat.rb

```diff
@@ -43,8 +45,11 @@ def install
     xy = Language::Python.major_minor_version "python3"
     ENV.prepend_create_path "PYTHONPATH", buildpath/"vendor/lib/python#{xy}/site-packages"
 
+    # Help Pillow find zlib
+    ENV.append_to_cflags "-I#{MacOS.sdk_path}/usr/include"
```

Comment:
In theory at least, this should not be necessary for Pillow 8.0, thanks to https://github.com/python-pillow/Pillow/pull/4974